app/views/embeddable/xhtmls/edit.html.haml

```diff
@@ -11,6 +12,15 @@
   .options
     = field_set_tag "Options" do
       .ul
+        .li
```

Comment:
I'm not that familiar with this form. But I think it'd be better if the heading field was nested under the is callout box. So it is more clear that checking the box will enable the header.

Reply:
We decided to allow a header even when the item is a callout. That simplifies this form and gives the authors more options.